lti_consumer/lti_xblock.py

```diff
@@ -100,6 +100,7 @@
     'staff': 'Administrator',
     'instructor': 'Instructor',
 }
+CUSTOM_PARAMETER_TEMPLATE_TAG = '&'
```

Comment:
@schenedx, I am using a pair of tags which are: **${** and **}** to wrap the **template_value** I checked in the LTI docs that they use **$** sign as the prefix for custom substitution, and in order to avoid collision with it we can use the pair of tags mentioned before, what do you think about it?